pkg/api/types.go

```diff
@@ -260,6 +263,9 @@ type PersistentVolumeSource struct {
 	Cinder *CinderVolumeSource `json:"cinder,omitempty"`
 	// CephFS represents a Ceph FS mount on the host that shares a pod's lifetime
 	CephFS *CephFSVolumeSource `json:"cephfs,omitempty"`
+	// FC represents a Fibre Channel resource that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	FC *FCVolumeSource `json:"fc,omitempty"`
```

Comment:
maybe I missed the debate, but 'fc' as part of the API is pretty obscure (as is 'rbd').  Is that really the best name?